test/ussd_chw.test.js

```diff
@@ -169,6 +163,74 @@ describe("app", function() {
         });
         // end re-dial flow tests
 
+        describe("dialback sms testing", function() {
+            it("send if redial sms not yet sent (identity loads without redial_sms_sent defined)", function() {
+                return tester
+                .setup.user.addr("27820001001")
+                .inputs(
+                    {session_event: 'new'}  // dial in
+                    , "1"  // state_start - yes
+                    , "1"  // state_consent - yes
+                    , {session_event: 'close'}
+                )
+                .check.user.answer("redial_sms_sent", true)
+                .check(function(api) {
+                    utils.check_fixtures_used(api, [124, 160, 163]);
+                })
+                .run();
+            });
+            it("don't send if redial sms already sent (identity loads with redial_sms_sent set as 'true')", function() {
+                return tester
+                .setup.user.addr("27820001010")
+                .inputs(
+                    {session_event: 'new'}  // dial in
+                    , "1"  // state_start - yes
+                    , "1"  // state_consent - yes
+                    , {session_event: 'close'}
+                )
+                .check.user.answer("redial_sms_sent", true)
+                .check(function(api) {
+                    utils.check_fixtures_used(api, [185]);
+                })
+                .run();
+            });
+            it("send if redial sms not yet sent (identity loads with redial_sms_sent set as 'false')", function() {
+                return tester
+                .setup.user.addr("27820001008")
```

Comment:
This identity only has it set to `false` for `nurseconnect` - it should be `false` for all the lines you're testing it against.